docs/book/orders.rst

```diff
@@ -70,8 +70,8 @@ Assuming that you have a **Product** with a **ProductVariant** assigned already
     /** @var ProductInterface $product */
     $product = $this->container->get('sylius.repository.product')->findOneBy([]);
```

Comment:
In this example it will be better to use `product_variant` repository and remove the below line.